tools/run-mypy

```diff
@@ -24,29 +24,31 @@ zproject/test_settings.py
 """.split()
 
 parser = argparse.ArgumentParser(description="Run mypy on files tracked by git.")
-parser.add_argument('targets', nargs='*', default=[],
+parser.add_argument('targets', nargs='*',
                     help="""files and directories to include in the result.
                     If this is not specified, the current directory is used""")
-parser.add_argument('-m', '--modified', action='store_true', default=False, help='list only modified files')
-parser.add_argument('-a', '--all', dest='all', action='store_true', default=False,
+parser.add_argument('-m', '--modified', action='store_true',
+                    help='list only modified files')
```

Comment:
For consistency, perhaps amend this to a triple-quote?

Reply:
Agree on the consistency -- a bit later in the series I amend them all to single double-quotes. :-)

(Single because they should fit on a line in the output, not be paragraphs; double because I weakly prefer that for strings that are text for the user.)